libsolidity/CompilerStack.h

```diff
@@ -75,7 +76,13 @@ class CompilerStack: boost::noncopyable
 	/// @returns the compiled bytecode
 	bytes const& compile(std::string const& _sourceCode, bool _optimize = false);
 
+	/// Gets the assembled bytecode for a contract
 	bytes const& getBytecode(std::string const& _contractName = "") const;
+	/// Get the runtime context's bytecode for a contract
```

Comment:
What about "@returns the runtime bytecode for the contract, i.e. the code that is returned by the constructor."?